docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+# Kubernetes on CenturyLink Cloud
```

Comment:
Add the following to get an in-page TOC since this page has a lot of sections:

* TOC
{: toc}